unittests/tools/test_snyk_parser.py

```diff
@@ -182,3 +182,15 @@ def test_snykcode_issue_9270(self):
             parser = SnykParser()
             findings = parser.get_findings(testfile, Test())
             self.assertEqual(39, len(findings))
+
+    def test_snykcode_issue_9270_epss(self):
+        with open("unittests/scans/snyk/snykcode_issue_epss.json") as testfile:
```

Comment:
`snykcode_issue_epss.json` vs `snykcontainer_issue_epss.json`

Reply:
Thanks for the notice!